core/spec.md

```diff
@@ -864,11 +906,13 @@ and the following Registry entity specific attributes:
 - Type: String
 - Description: The version of this specification that the serialization
   adheres to
-  TODO: "serialization" or "Registry"? Can a reg support more than one version?
 - Constraints:
-  - REQUIRED
-  - MUST be a non-empty string
+  - REQUIRED in responses, OPTIONAL in requests
+  - REQUIRED in document view
   - MUST be a read-only attribute in API view
+  - MUST be a non-empty string
```

Comment:
Just "MUST be non-empty"? (The fact that it's a string is indicated by the type...)

Reply:
fixed